index.html

```diff
@@ -0,0 +1,165 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://cdn.tailwindcss.com"></script>
+    <title>Document</title>
+</head>
+<body> 
+    <section class="text-gray-600 body-font relative bg-cyan-100">
+        <div class="container px-5 py-24 mx-auto">
+          <div class="flex flex-col text-center w-full mb-12">
+            <h1 class="sm:text-3xl text-2xl font-medium title-font mb-4 text-gray-900">Microverse Survey form</h1>
+            <p class="lg:w-2/3 mx-auto leading-relaxed text-base">Thank you for taking the time to help us improve the platform</p>
+          </div>
+          <div class="lg:w-1/2 md:w-2/3 mx-auto p-4 text-white" style="background-color: rgba(27, 27, 50, 0.8);">
+            <div class="flex flex-wrap -m-2">
+                <div class="p-2 w-full">
+                    <div class="relative">
+                      <label for="email" class="leading-7 text-sm">Name</label>
+                      <input type="text" id="name" name="name" placeholder="Enter your name" class="w-full text-gray-600 bg-opacity-50 rounded border border-gray-300 text-base outline-none py-1 px-3 leading-8 ">
+                    </div>
+                </div>
+                <div class="p-2 w-full">
+                    <div class="relative">
+                      <label for="email" class="leading-7 text-sm">Email</label>
+                      <input type="email" id="email" name="email" placeholder="Enter your email" class="w-full text-gray-600 bg-opacity-50 rounded border border-gray-300 text-base outline-none py-1 px-3 leading-8">
+                    </div>
+                </div>
```

Comment:
- [ ] Good job giving a comment 👍, however, as the previous reviewer said your suggestion is not detailed enough to help the student clearly understand what changes should be applied. Please kindly clearly state where the student needs to apply the changes for semantic purposes. An example could be changing a div to h1 and explaining clearly why the changes should be applied. Meaning, you should try as much as possible to avoid generalizing 👍.

Reply:
Hi..
please can i delete that commet because the project already has a good semantic html?

Reply:
Good job so far! 
As mentioned by the previous reviewer, your suggestion is not detailed enough to help the student clearly understand what changes should be applied. Please kindly clearly state where the student needs to apply the changes for semantic purposes. An example could be changing a div to h1 and explaining clearly why the changes should be applied. Meaning, you should try as much as possible to avoid generalizing 👍.

Kindly I suggest you approve the project if it follows HTML best practices and have good semantics.
Happy review! 😄